src/modules/jsUtils.js

```diff
@@ -0,0 +1,94 @@
+/**
```

Comment:
I had initially placed these functions in `utils.js`, but I had runtime errors: `Error: Cannot find module 'exception'`. Because there is no `exception` dependency in our `package.json`, I assume this means that the current `utils` module was meant to be run in a `browserify`'d environment, which is not the case for the server code.

Because of that, I thought the best approach was to have a separate `jsUtils` (or would `serverUtils` be better?) module that would not require to be inside a `browserify`'d environment to work.

@albertinad @dlebu what do you think? Would there be an alternative to having 2 separate util modules (one for `browserify`, and one for pure standalone JS)?